### DISCUSSION TITLE
I want to apply Github Education.but I'm on summer vacation and I'm going home

### DISCUSSION THREAD

**Discussion by tinysouth** [community/community#135905] [GitHub Education]
### Hi everyone,

I want to apply Github Education.but I'm on summer vacation and I'm going home
![Uploading Screenshot_2024-08-15-23-52-32-358_org.mozilla.firefox-edit.jpg…]()

**Comment by tinysouth** [comment]
I am on vacation but it shows that 
![Uploading IMG_20240816_001346.jpg…]()

**Comment by tinysouth** [comment]
What happened?
Suggested proof to expedite your approval:

    You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

**Comment by tinysouth** [comment]
You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

**Comment by davevad93** [comment]
Hi @zhouwei333 ,

If you aren't a distance learning student, you must be close to your school in order to apply. If your home location isn't close to your school, you'll have to wait until classes resume to apply for your student benefits and avoid being rejected.
